test/services/fileStorage/index.test.js

```diff
@@ -83,4 +84,26 @@ describe('fileStorage service', function () {
 
 		return Promise.all(promises);
 	});
+
+	it('should not allow any of these folders', () => {
+
+		let folderNames = [];
+
+		let promises = ['C_drive', 'Windows', '.3T', '$WINDOWSBD', ' ', 'k_drive', 'Temporary Items'];
+
+		folderNames.forEach(name => {
```

Comment:
`folderNames` is empty, so this won't do anything :-)

Reply:
also, why not use `const promises = folderNames.map((name) => ...)`

Reply:
sure, should have been the other way around.
I rather stay consistent with the rest of the code :-), it also has no advantage especially tests.